hack/install-etcd.sh

```diff
@@ -23,10 +23,6 @@ fi
 
 # setup a private GOPATH so the build can succeed
 export GOPATH="${PWD}/gopath"
-rm -f "${GOPATH}/src/github.com/coreos/etcd"
-mkdir -p "${GOPATH}/src/github.com/coreos"
-ln -s "${PWD}" "${GOPATH}/src/github.com/coreos/etcd"
```

Comment:
etcd ./build does this now - they didn't used to control their gopath, now they do